root/utility/tableColumns.js

```diff
@@ -342,6 +353,31 @@ export function defineReleaseLabelsColumn(
   };
 }
 
+export function defineRemoveFromMergeColumn(
+  toMerge: $ReadOnlyArray<CoreEntityT>,
+): ColumnOptions<ArtistT | RecordingT | ReleaseT, number> {
+  return {
+    Cell: ({row: {original}}) => {
+      const url = ENTITIES[original.entityType].url;
+      return toMerge.length > 2 ? (
+        <td>
+          <a href={`/${url}/merge?remove=${original.id}&submit=remove`}>
+            <button
+              className="remove-item icon"
+              title={l('Remove from merge')}
+              type="button"
+            />
+          </a>
+        </td>
+      ) : null;
+    },
+    Header: toMerge.length > 2 ? (
+      <th aria-label={l('Remove from merge')} style={{width: '1em'}} />
+    ) : null,
```

Comment:
Might be worth mentioning this code is being moved from  `root/components/RemoveFromMergeTableColumn.js` and `root/components/RemoveFromMergeTableHeader.js`, for example by inserting into these files a comment such as:
```js
// Converted to react-table at root/utility/tableColumns.js
```

Reply:
I'm removing these two files with the follow-up PRs anyway - one already with the artist PR, the other as soon as I get search working so I can get rid of ArtistListEntry :)

Reply:
Still, such comment or a note in commit message would make easier to track code in git history.

Reply:
By the way, why duplicating `RemoveFromMergeTableCell` for `Cell`?

Reply:
Because that file will also be deleted as soon as possible.

Reply:
Added that comment, anyway.

Reply:
For the record, added in https://github.com/metabrainz/musicbrainz-server/pull/1414/commits/291537028369ee52173e335118d31c918c7e8cc1.